File: uke0/bil-objekter-117/Bil.java
class Bil {
    private String eier;
    private String merke;

    public Bil(String eieren, String merket) {
	eier = eieren;
	merke = merket;
    }

    public void skrivInfo() {
	System.out.println("Eier av bilen er " + eier +
			   " og merket er " + merke + ".");
    }
}

File: uke0/bil-objekter-117/c++/Bil.cpp
#include "Bil.h"
#include <iostream>

Bil::Bil(const char* navn, const char* merke) : m_navn(navn), m_bilmerke(merke) { }
void Bil::skrivInfo()
{
  std::cout << "Eieren av bilen er " << m_navn;
  std::cout << " og merket er " << m_bilmerke << "." << std::endl;
}
